integration/metrics_test.go

```diff
@@ -153,10 +153,16 @@ log_fuse_operations = true
 
 	manipulateZtocMetadata := func(zt *ztoc.Ztoc) {
 		for i, md := range zt.FileMetadata {
-			md.UncompressedOffset += 2
-			md.UncompressedSize = math.MaxInt64
-			md.PAXHeaders = map[string]string{"foo": "bar"}
-			zt.FileMetadata[i] = md
+			/*
+				Changing UncompressedSize to be a large value results in a readdirent:value too large for defined data type error.
+				The aim of this test is to verify that fuse operations fail if ztoc data is manipulated, this can be achieved by modifying regular files.
```

Comment:
nit: A bit too verbose IMO, probably a one-sentence explanation will do (e.g. "We only want to manipulate regular files to avoid issues with older kernels" or something roughly as concise).